.travis.yml

```diff
@@ -1,7 +1,12 @@
 language: java
 sudo: false
+## export GPG details
 before_install:
-  - wget https://github.com/sormuras/bach/raw/master/install-jdk.sh
+  wget https://github.com/sormuras/bach/raw/master/install-jdk.sh;
+  if [[ "$TRAVIS_BRANCH" == "master" ]] && [[ -n $TRAVIS_TAG ]]; then
```

Comment:
Same here about opposite condition.

Reply:
Actually, what's the harm in always doing this?

Reply:
This was due to the master branch constraint on the Travis CI environment variables. Since I'm updating that, I agree, no harm. Thanks!